packages/local_auth/ios/Classes/FLTLocalAuthPlugin.m

```diff
@@ -130,6 +136,48 @@ - (void)authenticateWithBiometrics:(NSDictionary *)arguments
   }
 }
 
+- (void)authenticate:(NSDictionary *)arguments withFlutterResult:(FlutterResult)result {
+  LAContext *context = [[LAContext alloc] init];
+  NSError *authError = nil;
+  _lastCallArgs = nil;
+  _lastResult = nil;
+  context.localizedFallbackTitle = @"";
```

Comment:
Why is fallback being disabled here? That seems undesirable for the non-biometric-only case.

Reply:
Please escalate this!